docs/src/getting-started.md

```diff
@@ -3,14 +3,31 @@
 
 ## Basics
 
-[GraphQL.org](http://graphql.org/learn) is the best place to get started learning GraphQL.  Here is an excerpt from the introduction:
+As its name suggests, this library is a .NET implementation of [GraphQL](http://graphql.org/learn). This document assumes you are already familiar with the fundamentals of GraphQL.
 
-> GraphQL is a query language for your API, and a server-side runtime for executing queries by using a type system you define for your data. GraphQL isn't tied to any specific database or storage engine and is instead backed by your existing code and data.
+```
++------------------------------------+
+|           Interactivity            |
++------------------------------------+
++------------------------------------+
+|         +-----------------------+  |
+|         | Queries and Mutations |  |
+| GraphQL +-----------------------+  |
+|         | Types and Interfaces  |  |
+|         +-----------------------+  |
++------------------------------------+
++------------------------------------+
+|              Database              |
++------------------------------------+
+```
+
+GraphQL sits between your database the "outside world." This library only concerns itself with this middle layer. For simplicity, the documentation uses hard-coded data, but in a real-world application your database layer would consist of some sort of ORM, like Entity Framework. And the only aspect of interactivity we will discuss here is how to actually execute the query. How specifically the query is received and the output returned is beyond the scope of this document. See the ["Further Reading"](#further-reading) section at the end of this document for examples and tutorials specific to various ORMs and deployment scenarios.
```

Comment:
I think this overstates the case a bit. Graphql occupies the same same space as a REST api, that may mean the level below it is a database, but more often than not its probably a service layer or other business logic  specfic thing between the DB or ORM. It just talk to the ORM directly but we shouldn't imply that's the only or correct way to use it.

Reply:
We can certainly expand on that. Any suggestions? 

Would you prefer calling it a more general "data layer" instead of "database"? Something like:

> GraphQL sits between your data and the "outside world." ...but in a real-world application your data layer would consist of an ORM or some other data service provider...